virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -24,48 +32,32 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 [NOTE]
 ====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
+There are multiple methods for configuring a VLAN, for example, creating a node network configuration policy. However, a network attachment definition provides a more efficient and manageable configuration
```

Comment:
I thought about this some more. I think we can make it more concise. Suggestion:

A Linux bridge network attachment definition is the most efficient method for configuring a VLAN interface and provides the most control.

I think we don't need to say that multiple methods exist because that is self-evident. It's also implied by "most efficient method". What do you think?

Reply:
I do like your rewording as it's definitive --- and telling them there are multiple methods really does have NO value when you think about it (likely they know this anyway). So I made that change to both the CLI and UI modules.